src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -852,7 +926,9 @@ def _create_unsuccessful_markdown(run_results: dict) -> str:
                 n.node.resource_type,
                 n.message,
                 n.node.path,
-                n.node.compiled_code if not n.node.resource_type == "seed" else None,
+                n.node.compiled_code
+                if n.node.resource_type not in ["seed", "source"]
+                else None,
```

Comment:
Could you clarify why source nodes are excluded from artifact generation here?

Reply:
Hi @zzstoatzz, dbt source freshness command result has node type of `SourceDefinition`, which does not have compiled_code property, thus the exclusion. I wanted to make it explicit since n.node does not have a type and could lead to someone accidentally call a string method on it or something. Functionally, this code does the same as the old one since `n.node.compiled_code` is always None.